src/bepasty/apis/__init__.py

```diff
@@ -1,6 +1,13 @@
 from flask import Blueprint
 
+from .lodgeit import LodgeitUpload
+from .rest import ItemDetailView, ItemDownloadView, ItemUploadView, InfoView
+
+
 blueprint = Blueprint('bepasty_apis', __name__, url_prefix='/apis')
 
-from . import (
-    lodgeit, rest)
+blueprint.add_url_rule('/lodgeit/', view_func=LodgeitUpload.as_view('lodgeit'))
+blueprint.add_url_rule('/rest', view_func=InfoView.as_view('api_info'))
+blueprint.add_url_rule('/rest/items', view_func=ItemUploadView.as_view('items'))
+blueprint.add_url_rule('/rest/items/<itemname:name>', view_func=ItemDetailView.as_view('items_detail'))
+blueprint.add_url_rule('/rest/items/<itemname:name>/download', view_func=ItemDownloadView.as_view('items_download'))
```

Comment:
hmm, this is moving around quite a lot of stuff just to get flake8 silent.

i dislike a bit that the rules are not added in the module where the functionality is implemented, but everything is moved up to init.

Reply:
Well, and I can say that it's strange to add them to ***blueprint*** in place other than definition of blueprint. Because of that (creating blueprint in one place and extending it in other place) you needed to split those imports to two parts to avoid circular imports. That was a signal, you had something wrong with project structure. IMO.

Reply:
ok, you convinced me. :)